src/main/java/com/example/demo/controllers/ItemController.java

```diff
@@ -0,0 +1,51 @@
+package com.example.demo.controllers;
+
+import java.util.Collection;
+
+import javax.annotation.PostConstruct;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.example.demo.models.ItemModel;
+import com.example.demo.models.UserModel;
+import com.example.demo.repositories.ItemsRepository;
+import com.example.demo.services.ItemService;
+import com.example.demo.services.interfaces.IItemService;
+
+@RestController
+public class ItemController {
+	@Autowired
+	ItemsRepository itemsRepo;
+	
+	IItemService itemService;
+	
+	@PostConstruct
+	public void init() {
+		itemService=new ItemService(itemsRepo);
+	}
+	
+	@GetMapping("/api/items/{page}/{count}")
```

Comment:
we should switch to query paramters for this case /api/items?page=0&count=10

this gives us also some code reusability because we can have PaginatedParams that handles this data for all endpoints